docs/operations/dex.md

```diff
@@ -6,7 +6,7 @@ detailed below.
 
 ### 1. Register application for Github
 
-In Github, register a new application named `Numaflow Server App`. The callback address should be the
+In Github, register a new OAuth application named `Numaflow Server App`. The callback address should be the
```

Comment:
nit: the application name doesn't have to be `Numaflow Server App`, right? Users can choose their name.

Reply:
Going to run a test with this now. As it stands, the static client that is written to the dexConfig is:
```
numaflowStaticClient := map[string]interface{}{
		"id":     "numaflow-server-app",
		"name":   "Numaflow Server App",
		"public": true,
		"redirectURIs": []string{
			redirectURL,
		},
	}
```

Reply:
The name of the Oauth shouldn't matter since we use client_id and client_secret to configure the connector. If the system is implemented in a way that user has to name "Numaflow Server App", then we should change the implementation...

Reply:
Works fine without having to name it `Numaflow Server App`. Will change this!